### PR TITLE
Fixes mechanical surrendering, and makes it more visible

### PR DESCRIPTION
**Pull request by CalamaBanana**
## About The Pull Request
image

Did you know we had mechanical surrendering in the game? 
Right-click combat mode to wave a little flag and be frozen for a bit. Held items will be drop. You can still lie down, but not move.
The flag is waving.

My changes:
- Fixed bug in the hotkey for it, now it actually works. Default is and was Shift+X. 
- Disabled useless safety popup. No one even knew that popup existed. If I had not written this, you most likely wouldn't either.
- Doubled the size of the flag sprite, and made it more noticeable. 

Tip: Despite being frozen, you can still lie down while waving your flag. 
image

## Why It's Good For The Game

No one even knew this existed. People do keep clicking sprites till they stop moving because there was no known way to signal that your sprite is willing to do that without dying. So I fixed up what was already there.

This will lead to more surrender, more captives. More RP. Not just headless sprites all around.

### CODE REVIEW

**Comment by Hmeister-real** [comment]
this is cool, but it's a little emmersion breaking for a random UI element to pop up over someone, when nothing else does that as of now.

**Comment by CalamaBanana** [comment]
> this is cool, but it's a little emmersion breaking for a random UI element to pop up over someone, when nothing else does that as of now.

That is kinda the point. Get someones attention to stop clicking the sprite. There are other UI elements that can pop up, too. You just rarely see them.
As I said, I did create this. I just fixed it up.



> I wonder if RMB on Combat Mode should be removed if the hotkey works, to prevent people accidently surrendering

Find me one person who had actually seen that popup ingame. I could not. I strongly believe this is a non-issue.

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -1042,17 +1042,17 @@
...
-		var/image/flaggy = image('icons/effects/effects.dmi',src,"surrender",ABOVE_MOB_LAYER)
-		flaggy.appearance_flags = RESET_TRANSFORM|KEEP_APART
-		flaggy.transform = null
-		flaggy.pixel_y = 12
-		flick_overlay_view(flaggy, src, 150)
-		Stun(150)
-		src.visible_message(span_notice("[src] yields!"))
-		playsound(src, 'sound/misc/surrender.ogg', 100, FALSE, -1)
-		sleep(150)
+	//if(alert(src, "Yield in surrender?",,"YES","NO") == "YES")
```
[the remark this answers is not in the document]
Comment (CalamaBanana) [reply code/modules/mob/living/living.dm]: Gotcha

**Comment by CalamaBanana** [comment]
> Is it too hard for people to just scream "I YIELD". Personally I feel like this mechanic could be removed altogether.

The typing is easy. It being seen in a full courtroom of stabbing and murder not. And this mechanic gives the other side the security that one is truly surrendering.

File: code/datums/keybinding/living.dm [hunk 2]
```diff
@@ -253,4 +253,4 @@
 		lastrest = world.time
 		return TRUE
 	else
-		return FALSE
\ No newline at end of file
+		return FALSE
```
[the remark this answers is not in the document]
Comment (CalamaBanana) [reply code/datums/keybinding/living.dm:256]: Any deeper reason for it than it looking neat? I did not touch byond code in years.

**Comment by CalamaBanana** [comment]
Yes, absolutely. This is just the equivalent of holding up your hands (or getting into the position). Up to the other side if he keeps stabbing or not.

**Pull request merged**